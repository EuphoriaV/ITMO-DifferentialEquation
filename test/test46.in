2.33
(((((x * x) * (x * x)) * (x - x)) * (((x - 8.24) - (x * 2.62)) * (x - x))) + ((x * x) * (x - 4.06)))
3.16
3.25