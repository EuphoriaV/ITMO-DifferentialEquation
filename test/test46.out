7.898E-4